docker/conf/docker-compose.github-actions.review.yml.dist

```diff
@@ -111,6 +111,39 @@ services:
             - chadburn.job-exec.BRANCH_NAME-cron-packetery.user=www-data
             - chadburn.job-exec.BRANCH_NAME-cron-packetery.tty=true
 
+    php-consumer:
+        restart: always
+        image: php-fpm-image
+        volumes:
+            - web-volume:/var/www/html/project-base/app/web
+        depends_on:
+            - php-fpm
+            - rabbitmq
+        environment:
+            IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK: 1
+            FORCE_ELASTIC_LIMITS: true
+            GOPAY_EN_GOID: GOPAY_GOID
+            GOPAY_EN_CLIENTID: GOPAY_CLIENTID
+            GOPAY_EN_CLIENTSECRET: GOPAY_CLIENTSECRET
+            GOPAY_CS_GOID: GOPAY_GOID
+            GOPAY_CS_CLIENTID: GOPAY_CLIENTID
+            GOPAY_CS_CLIENTSECRET: GOPAY_CLIENTSECRET
+            PACKETERY_API_KEY: PACKETERY_API_KEY_VALUE
+            PACKETERY_API_PASSWORD: PACKETERY_API_PASSWORD_VALUE
+        networks:
+            - default
+        entrypoint:
+            - /bin/sh
+            - -c
+            - sleep 5 && project-base/app/docker/php-fpm/consumer-entrypoint.sh 600
```

Comment:
Why this needs to be defined differently for review server?

Reply:
Because the review server does not change its source code, it's okay to let the consumer run longer. 
sleep is added to give other services time to initialize and become available before the consumer script starts executing. It's not necessary in the local environment